src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-snapshotschedule-list/cephfs-snapshotschedule-list.component.html

```diff
@@ -4,37 +4,64 @@
   </cd-loading-panel>
 </ng-container>
 
-<ng-template #pathTpl
-             let-row="row">
+<cd-alert-panel
+  *ngIf="(snapScheduleModuleStatus$ | async) === false"
+  type="info"
+  spacingClass="mb-3"
+  i18n
+  class="d-flex align-items-center"
+>
+  In order to access the snapshot scheduler feature, the snap_scheduler module must be enabled
+  <button
+    class="btn btn-light mx-2"
+    type="button"
+    (click)="enableSnapshotSchedule()">
+    Enable
+  </button>
+</cd-alert-panel>
+
+<ng-template
+  #pathTpl
+  let-row="row">
   <span
     class="fw-bold"
     [ngbTooltip]="fullpathTpl"
-    triggers="click:blur">{{row.path | path}}</span>
+    triggers="click:blur">
+    {{ row.path | path }}
+  </span>
 
-  <span *ngIf="row.active; else inactiveStatusTpl">
-    <i [ngClass]="[icons.success, icons.large]"
-       ngbTooltip="{{row.path}} is active"
-       class="text-success"></i>
+  <span
+  *ngIf="row.active; else inactiveStatusTpl">
+    <i
+      [ngClass]="[icons.success, icons.large]"
+      ngbTooltip="{{ row.path }} is active"
+      class="text-success"
+    ></i>
```

Comment:
looks like this is just here because of the html formatting which is unnecessary!

Reply:
Can you please elaborate your comment? This markup is part of the template and it is necessary to display the tooltip and the icon according to the snap schedule status.

Reply:
No, I was saying that there is no change happening to this area apart from what looks like a linter being configured differently. If its just that, can we avoid changing these?